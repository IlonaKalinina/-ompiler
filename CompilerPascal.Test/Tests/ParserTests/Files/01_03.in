const
a = 7;
b = 1.2e3;
begin
end.