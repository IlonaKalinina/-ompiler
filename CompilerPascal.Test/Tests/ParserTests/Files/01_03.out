program 
├───const
│    ├─── =
│    │    ├─── a
│    │    └─── 7
│    └─── =
│         ├─── b
│         └─── 1200
│    
begin
└─── end
